packages/boot/test/bootstrapTests/liquidation.ts

```diff
@@ -350,8 +350,7 @@ const addSTARsCollateral = async (
 
   t.log('building proposal');
   const proposal = await buildProposal({
-    package: 'builders',
-    packageScriptName: 'build:add-STARS-proposal',
+    builderPath: '@agoric/builders/scripts/inter-protocol/add-STARS.js',
   });
```

Comment:
now that it's one param it could be,
```js
const proposal = await buildProposal('@agoric/builders/scripts/inter-protocol/add-STARS.js');
```
but there's another named param it still takes `env` and I'm not sure what that's for. Not a blocker but it would be good to document what it's for or at least have some code that uses it.